Foundation/URLSession/TransferState.swift

```diff
@@ -83,8 +83,16 @@ extension _HTTPURLProtocol._TransferState {
     /// return value's `isHeaderComplete` will then by `true`.
     ///
     /// - Throws: When a parsing error occurs
-    func byAppending(headerLine data: Data) throws -> _NativeProtocol._TransferState {
-        guard let h = parsedResponseHeader.byAppending(headerLine: data) else {
+    func byAppendingHTTP(headerLine data: Data) throws -> _NativeProtocol._TransferState {
+	 /// If the line is empty, it marks the end of the header, and the result
```

Comment:
This should be a comment, not a doc comment. Please also fix the indentation for `- Note:`.